.appveyor.yml

```diff
@@ -0,0 +1,22 @@
+version: 3.3.0.{build}
+
+skip_branch_with_pr: true
+os: Visual Studio 2019
+
+configuration:
+  - Debug
+  - Release
+
+environment:
+  WTG_ANALYZERS_VERSION: 3.3.0
```

Comment:
Do we still need this? At best we should probably rename it.

Reply:
Good point, I'll remove it.